doc/modules/computing.rst

```diff
@@ -504,20 +504,140 @@ Links
   - `Scipy sparse matrix formats documentation <https://docs.scipy.org/doc/scipy/reference/sparse.html>`_
 
 Parallelism, resource management, and configuration
-=====================================================
+===================================================
 
 .. _parallelism:
 
-Parallel and distributed computing
------------------------------------
+Parallelism
+-----------
 
-Scikit-learn uses the `joblib <https://joblib.readthedocs.io/en/latest/>`__
-library to enable parallel computing inside its estimators. See the
-joblib documentation for the switches to control parallel computing.
+Scikit-learn supports parallel implementations for estimators and other
+utilities in two ways:
+
+- via the `joblib <https://joblib.readthedocs.io/en/latest/>`_ library. In
+  this case the number of threads or processes can be controlled with the
+  ``n_jobs`` parameter.
+- via OpenMP, used in C or Cython code.
+
+In addition, some of the numpy routines that are used internally by
+scikit-learn may also be parallelized if you rely on specific numerical
+libraries such as MKL, OpenBLAS, or BLIS.
+
```

Comment:
```suggestion
scikit-learn may also be parallelized if numpy is installed with specific
numerical libraries such as MKL, OpenBLAS or BLIS
```

I find it clearer to say that this mainly depends on the way `numpy` is installed.